.golangci.yaml

```diff
@@ -0,0 +1,25 @@
+run:
+  # Default timeout is 1m, up to give more room
+  timeout: 4m
+
+linters:
```

Comment:
This can also be done as a follow-up: it might be worth expanding the `importas` linter and specifying a core of common package imports we expect, e.g. `rukpakv1alpha1`, `corev1`, `metav1`, etc. You can find how to do that using [golangci-lint's documentation around the importas linter](https://golangci-lint.run/usage/linters/#importas).